llvm/test/lab2/kostanyan_arsen/tests.ll

```diff
@@ -0,0 +1,111 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/InstrumentFunc_Kostanyan_Arsen_FIIT2%pluginext\
```

Comment:
Could you please also add tests:
- input llvm ir without predefined instrument_* functions.
- function with already inserted instrument_* calls.